app/views/bus_stops/by_route.haml

```diff
@@ -3,9 +3,14 @@
 - if @stops.empty?
   No stops on this route
 - else
-  %ul
-    - @stops.each do |stop|
+  - @inc_and_comp.each_pair do |complete, stops|
```

Comment:
`@inc_and_comp` is equivalent to `@stops.group_by(&:completed?)`, so `@stops.empty?` is fine to check that `@inc_and_comp` is empty.